src/main/java/org/craftercms/studio/model/rest/content/RenameRequestBody.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2007-2022 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.model.rest.content;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.validation.constraints.NotEmpty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Do we want to ignore unknown invalid properties? Several tickets have been reported requiring to remove ignoreUnknown param
cc: @Euquimides

Reply:
Removed `@JsonIgnoreProperties(ignoreUnknown = true)` notation. Thank you!